test/e2e/infra/secondary_network/secondary_network_prerequisite.sh

```diff
@@ -0,0 +1,360 @@
+#!/bin/bash
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+THIS_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+pushd $THIS_DIR
+
+_usage="Usage: $0 [--configure-prerequisites] [--deconfigure-prerequisites]
+                  [--base-interface <interface name>] [--sriov-vf-count <count>] [--host_name]
+
+Deploy Antrea CNI and its native secondary network prerequisites on an existing/running K8s cluster control node.
+
+        --configure-prerequisites     Deploy all the pre-requisite plugins and CRDs for secondary network configuration.
+        --deconfigure-prerequisites   Remove all the pre-requisite plugins and CRDs deployed for secondary network configuration.
```

Comment:
maybe rename these to `--setup` and `--cleanup`

Reply:
yes. done.